src/modules/git_status.rs

```diff
@@ -80,33 +57,47 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
 
     // Add the conflicted segment
     if let Ok(repo_status) = repo_status {
-        if repo_status.is_conflicted() {
-            module.new_segment("conflicted", GIT_STATUS_CONFLICTED);
-        }
+        create_segment_with_count(
+            &mut module,
+            "conflicted",
+            repo_status.conflicted,
+            &config.conflicted,
+            config.conflicted_count,
+        );
     }
 
     // Add the ahead/behind segment
     if let Ok((ahead, behind)) = ahead_behind {
         let add_ahead = |m: &mut Module<'a>| {
-            m.new_segment("ahead", GIT_STATUS_AHEAD);
-
-            if show_sync_count {
-                m.new_segment("ahead_count", &ahead.to_string());
-            }
+            create_segment_with_count(
+                m,
+                "ahead",
+                ahead,
+                &config.ahead,
+                CountConfig {
+                    enabled: config.show_sync_count,
+                    style: None,
```

Comment:
why is the style None here?

Reply:
So we fallback to the module style and preserve existing behaviour.

A followup PR I'd like to propose: replace this option
* `show_sync_count: Bool` 

with:

* `ahead_count: CountConfig`
* `behind_count: CountConfig`

to make consistent with the new options in this PR and to allow styling the two counts.

Reply:
It's equivalent to config.style, as a `None` style will fall back to `module.style` when calling `module.create_segment`.

Reply:
I see